README.md

```diff
@@ -34,4 +34,17 @@ GitHub: https://github.com/Unyielding1
 Twitter: https://twitter.com/UnyieldingOne
 LinkedIn: https://www.linkedin.com/in/david-verde-3349b114b/
 
+
+## Acknowledgements
+
+ Cindy Shin (template creator)
+ 
+ https://linkedin.com/in/adagio07
+
+## Acknowledgements
+
+ Cindy Shin (template creator)
```

Comment:
[OPTIONAL]
- It would be awesome if you put a link reference to [Cindy Shin](https://www.behance.net/adagio07)